ci/prs.yml

```diff
@@ -159,6 +169,6 @@ jobs:
 
         user=$(user_slack_handle $(branch_sha))
         if [ "$user" != "" ]; then
-            tell_slack "<@${user}> <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for <https://github.com/digital-asset/daml/pull/$(pr.num)|PR $(pr.num)> has completed with status $(build_status) (changelog: $(changelog_status))." \
+            tell_slack "<@${user}> <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|Build $(Build.BuildId)> for <https://github.com/digital-asset/daml/pull/$(pr.num)|PR $(pr.num)> has completed with status $(build_status) (changelog: $(changelog_status), proto compat: $(check_compat_proto))." \
```

Comment:
I'm not sure we want this message to get longer. What is the value you expect to get from adding this here?

Reply:
Indeed, not really helpful. Will remove it.